.github/workflows/test-on-crc.yaml

```diff
@@ -0,0 +1,80 @@
+name: Test Chains on CRC
+on:
+  # only check PRs for new issues until preexisting issues are cleaned
+  pull_request:
+   branches:
+      - main
+  push:
+    branches:
+      - run-e2e-test-using-crc-in-github-actions
```

Comment:
@PuneetPunamiya can you change this to `on-push` on `main` branch instead ?

Reply:
Yeah, I'll update the pr with `on-push`

Reply:
Updated!